raco/myrial/emitarg.py

```diff
@@ -96,7 +96,7 @@ def get_statemods(self):
         return self.statemods
 
     def __repr__(self):
-        return 'NaryEmitArg(%r)' % self.sexprs
+        return 'NaryEmitArg(%r,%r)' % (self.column_names, self.sexprs)
```

Comment:
This is less broken than the original, but still broken. You need to include all constructor args:
```
return 'NaryEmitArg(%r,%r,%r)' % (self.column_names, self.sexprs, self.statemods)
```